src/types/react-table-config.ts

```diff
@@ -62,9 +62,22 @@ declare module 'react-table' {
   export type FieldType = 'text' | 'number' | 'date' | string;
 
   export type CellRendererProps<D extends object = {}> = {
+    /**
+     * Cell HTML props passed from the Table.
+     */
     cellElementProps: TableCellProps;
+    /**
+     * Table specific cell props like `column`, `row`.
+     */
     cellProps: CellProps<D>;
+    /**
+     * Cell's content.
+     */
     children: React.ReactNode;
+    /**
+     * Function that returns whether the cell is disabled.
+     */
+    isDisabled?: (rowData: D) => boolean;
```

Comment:
As we already have new minor release so I think it is ok but then it will be shown in changelog 😕

Reply:
Just change the message in changelog?

Reply:
that's the point, isn't it? it should show in changelog because it's a new addition

`feat(Table): Add `isDisabled` prop to `DefaultCell` and `cellRenderer`

Reply:
But my other PR would add normal message and examples. So do we need to bother with this?

Reply:
Not to make another PR too big and if something is stuck in another PR, it would delay CSS changes.

Reply:
ok, lets keep this if it's needed to avoid breaking disabled state